.buildbot.sh

```diff
@@ -238,12 +238,16 @@ for tracer in $TRACERS; do
 
         # Do a quick run of the benchmark suite as a smoke test.
         pipx install rebench
-        git clone https://github.com/ykjit/yk-benchmarks
+        git clone https://github.com/ykjit/yk-benchmarks --recurse-submodules
```

Comment:
Since we're here, we might as well add `--depth=1` to reduce network traffic.

Reply:
Right!

Reply:
I didn't know that was a thing, but I think I'm "yes"!

Reply:
Ah, didn't see this. One sec.